src/main/java/io/xpire/ui/HelpWindow.java

```diff
@@ -15,7 +15,8 @@
  */
 public class HelpWindow extends UiPart<Stage> {
 
-    public static final String USERGUIDE_URL = "https://se-education.org/addressbook-level3/UserGuide.html";
+    public static final String USERGUIDE_URL =
+            "https://github.com/AY1920S1-CS2103T-F11-2/main/blob/master/docs/UserGuide.adoc";
```

Comment:
I think this will link to the page on github. Try linking to our project website (github pages) instead, the directory should end with .html

Reply:
can